spras/domino.py

```diff
@@ -192,8 +192,10 @@ def parse_output(raw_pathway_file, standardized_pathway_file):
                     # columns that indicate edges
                     # Dropping the other rows eliminates the node information
                     module_df = pd.DataFrame(entries)
-                    module_df = module_df.loc[:, ['source', 'target']].dropna()
-
+                    try:
+                        module_df = module_df.loc[:, ['source', 'target']].dropna()
+                    except KeyError:
+                        module_df = pd.DataFrame()
```

Comment:
Just to confirm, this is creating an empty df because there are no source/target columns that indicate edges? And ere the potential empty cases are no source/target keys and/or source/target columns with no content?